frontend/src/components/SuperUser/SuperUserAdmin/Charts/GraphBarChart.jsx

```diff
@@ -0,0 +1,37 @@
+import {
+  Bar,
+  BarChart,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from "recharts";
+
+function GraphBarChart({ fetchTransitions }) {
+  const sellerSellData = fetchTransitions.reduce((acc, obj) => {
+    const sellerName = obj.seller_name;
+    acc[sellerName] = acc[sellerName] || { sellerName, totalSales: 0 };
+    acc[sellerName].totalSales += obj.total_amount_in_bhat;
+    return acc;
+  }, {});
+
+  const sellerData = Object.values(sellerSellData).map((sellerData) => ({
+    seller: sellerData.sellerName,
+    totalSales: sellerData.totalSales,
+  }));
+
+  return (
+    <div className="bar-chart">
+      <ResponsiveContainer width={"100%"} height={"100%"}>
```

Comment:
css need to see width to 100% and only change the parents container to responsive